dissect/target/plugins/apps/av/trendmicro.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+from typing import Iterator
+
+from dissect import cstruct
+
+from dissect.target import Target
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers.record import TargetRecordDescriptor
+from dissect.target.plugin import Plugin, export
+
+TrendMicroWFLogRecord = TargetRecordDescriptor(
+    "application/av/trendmicro/wflog",
+    [
+        ("datetime", "ts"),
+        ("string", "threat"),
+        ("string", "message"),
+        ("string", "keywords"),
+        ("string", "fkey"),
+    ],
+)
+
+
+TrendMicroWFFirewallRecord = TargetRecordDescriptor(
+    "application/av/trendmicro/wffirewall",
+    [
+        ("datetime", "ts"),
+        ("net.ipaddress", "local_ip"),
+        ("net.ipaddress", "remote_ip"),
+        ("string", "direction"),
+        ("uint16", "port"),
+        ("path", "path"),
+        ("string", "description"),
+    ],
+)
+
+
+c_pfwlog_i = """
+struct firewall_entry {
+    char      _[1];
+    char      direction;
+    uint16    port;
+    uint32    timestamp;
+    char      _pad[8];
+    char      local_ip[65];
+    char      remote_ip[65];
+    char      path[520];
+    wchar     description[128];
+    char      _[10];
+};
+"""
```

Comment:
```suggestion
pfwlog_def = """
struct firewall_entry {
    char      _[1];
    char      direction;
    uint16    port;
    uint32    timestamp;
    char      _pad[8];
    char      local_ip[65];
    char      remote_ip[65];
    char      path[520];
    wchar     description[128];
    char      _[10];
};
"""
c_pfwlog = cstruct.cstruct()
c_pfwlog.load(pfwlog_def)
```
Maybe do something more like this, instead of in the class init. This is more in line with how we use cstruct in other places, plus it saves a parse and compilation time on plugin initialization.